src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -83,72 +113,72 @@ def _get_declaration(
             for predicate_function, handle_function in list_of_predicate_and_handles:
                 if predicate_function(value.type):
                     attribute_value = handle_function(value.type)
-                    lines.append(f"{BASE_INDENT}{attribute_name}: {attribute_value}")
+                    default_value = _get_case_attr_default_value(value)
+
+                    # For Schema: if the attributes have a default value the entry is optional.
+                    if default_value and is_schema:
+                        attribute_value += " # optional"
+                        default_value = ""
+
+                    # If attribute_value ends with backslash (because of the cross-reference), add extra space
+                    # for a default value.
+                    # Otherwise, the equal symbol will be sticky with the link.
+                    if default_value and attribute_value[-1] == "\\":
+                        default_value = f" {default_value}"
+
+                    line_content = f"{BASE_INDENT}{attribute_name}: {attribute_value}{default_value}"
+
+                    # Remove the last character if it's a backslash otherwise the line break will be avoided.
```

Comment:
This entire block of code is more complex/cryptic than it should be. 😁 

I suggest:

```python
# Remove the last character if it's a backslash otherwise the line break will be avoided.
if line_content.endswith("\\"):
    line_content = line_content[:-1]
lines.append(line_content) 
```

Reply:
dammit, I thought that I was being clever 😅